x-pack/plugins/lens/public/vis_type_alias.ts

```diff
@@ -34,10 +34,11 @@ export const getLensAliasConfig = (): VisTypeAlias => ({
       searchFields: ['title^3'],
```

Comment:
This is weird, because I can definitely search the description when I type a unique string, but I think this is supposed to be:

```suggestion
      searchFields: ['title^3', 'description'],
```

Reply:
@wylieconlon you got me curious about this one – I investigated a bit. 

Turns out that the request doesn't use this param from here, but takes them from: 
https://github.com/elastic/kibana/blob/master/src/plugins/visualizations/public/saved_visualizations/find_list_items.ts#L55

We search through both lens and visualize so I think it's good . Example of request: 
`http://localhost:5603/clb/api/saved_objects/_find?default_search_operator=AND&page=1&per_page=1000&search=lens_vis*&search_fields=title%5E3&search_fields=description&type=lens&type=visualization`